### ISSUE TITLE
Decide if directional_offset_to should be added to SkyCoord

### ISSUE THREAD

**Issue by eteq** [astropy/astropy#8015]
In #5727 @dmopalmer  aimed to add two methods to `SkyCoord`: `directional_offset_by`and `directional_offset_to` - however, only the first of those got in for 3.1 - see my comment here: https://github.com/astropy/astropy/pull/5727#issuecomment-433678969

This issue is for discussing `directional_offset_to`. Should it be included in 3.2?  I'm specifically milestoning it for that as all that would be needed would be a PR to add back in the method that 1e1eba8654a5d2000037cfcbb63b0b78852b24ab deleted (and the associated test change).  So if we can come to consensus on it so be it.

Basically the core issue is this: in my view it's not necessary because `position_angle` and `separation` together do the same thing (with at *most* a ~10% speed loss in the scalar case), and "less is more"... Not to mention the similar name would mean it might get confused with the otherwise impossible-to-achieve `directional_offset_by`

cc @adrn @taldcroft @mhvk @hamogu (as participants in #5727 and the related #5702)

**Comment by eteq** [comment]
(pushing to 4.0 because no other opinions have been issued and as I said above I don't necessarily see the need for this)

**Comment by adrn** [comment]
@eteq and I just discussed and concluded this can be closed.
